micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,110 @@
+        viewActions={fetchActionItems(finalData)}
+      />
+      <Button
+        className={"mdms-view-audit"}
+        label="view audit"
+        variation="secondary"
+        icon={"History"}
+        onClick={() => {
+          history.push(`../utilities/audit-log?id=${finalData?.id}&tenantId=${tenantId}`)
+        }}
+      ></Button>
+      {showToast && <Toast label={t(showToast.label)} error={showToast?.isError} onClose={() => setShowToast(null)}></Toast>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)